client/src/css/stylesheet.scss

```diff
@@ -35,177 +42,315 @@ body {
-        z-index: 1;
 
-        #battery-icon {
+        #soc-icon {
+          width: 50%;
+          height: 100%;
           position: absolute;
-          top: 65%;
-          left: 10%;
-          width: 35%;
-          height: 35%;
+          z-index: 1;
+
+          #battery-icon {
+            position: absolute;
+            width: 35%;
+            height: 30%;
+            right: -20%;
+            bottom: 10%;
+          }
         }
-      }
 
-      #soc {
-        position: absolute;
-        top: 0%;
-        left: 0%;
-        z-index: 3;
-        height: 100%;
+        #soc {
+          position: absolute;
+          z-index: 3;
+          bottom: 17%;
+          right: -10%;
+          height: 75%;
+          width: 100%;
+
+          #soc-path {
+            fill-opacity: 0;
+            stroke-linecap: "butt";
+            stroke: #11b981;
+            stroke-opacity: 1;
+            stroke-width: 2.5rem;
+          }
+
+          #soc-text {
+            display: none;
+          }
+        }
       }
     }
 
-    #graph-wrapper {
+    #readout-wrapper {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: center;
+      align-content: space-around;
+      align-items: center;
+      width: 50%;
+      height: 88%;
+      top: 12%;
+      right: 0%;
       position: absolute;
-      right: 6%;
-      top: 15%;
-      width: 45%;
-      height: 45%;
 
-      #graph {
+      #graph-wrapper {
         position: relative;
-        height: 100%;
-        width: 100%;
+        width: 88%;
+        height: 45%;
 
-        .tick-label {
-          fill: white;
-        }
+        #graph {
+          position: relative;
+          height: 100%;
+          width: 100%;
 
-        .text-line {
-          fill: white;
-        }
+          .tick-label {
+            fill: white;
+            font-size: 1em;
+          }
+
+          .text-line {
+            fill: white;
+          }
+
+          .x-axis-tick .tick-label {
+            // Force the visibility on the x axis. The end tick labels are being
+            // overridden to visiblility: hidden in the Plottable JS library
+            // explicitly and I don't want to fiddle with the Plottable src.
+            visibility: visible !important;
+          }
 
-        .x-axis-tick .tick-label {
-          font-size: 0;
+          .x-label {
+            font-family: "Open Sans";
+            font-size: 1em;
+          }
+
+          .x-gridlines line {
+            stroke: #ccc;
+            stroke-width: 0.1em;
+          }
         }
       }
-    }
 
-    #state-wrapper {
-      position: absolute;
-      bottom: 0%;
-      left: 0%;
-      height: 15%;
-      width: 7%;
-      text-align: center;
+      #graph-spacer {
+        width: 12%;
+        height: 45%;
+      }
+
+      text {
+        font-family: "Open Sans";
+        text-anchor: middle;
+        alignment-baseline: middle;
+        fill: white;
+      }
+
+      #motor-wrapper {
```

Comment:
``body #dash-wrapper #readout-wrapper #motor-wrapper`` and ``body #dash-wrapper #readout-wrapper #solar-wrapper`` share these 3 declarations. Would it make sense to make a rule that applies these to both?

- height: 45%
- position: relative
- width: 45%